annotations.go

```diff
@@ -0,0 +1,213 @@
+package appoptics
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type AnnotationStream struct {
```

Comment:
Normally I'm not a huge stickler on this, but we should probably document everything that's exported for a nice godoc.

Reply:
(I meant to add since it's a public API/lib)

Reply:
will do